### PR TITLE
Fix for mismatched Mic network stream

### LINKED ISSUE #786: MicrophoneTransmitter / MicrophoneReciever network stream mismatch
In the MicrophoneTransmitter class the direction of the audio source is added to the network stream:
[MicrophoneTransmitter.cs#L274](https://github.com/Microsoft/HoloToolkit-Unity/blob/master/Assets/HoloToolkit/Sharing/Scripts/VoiceChat/MicrophoneTransmitter.cs#L274)

But in the MicrophoneReciever the code to extract this has been commented out:
[MicrophoneReceiver.cs#L258](https://github.com/Microsoft/HoloToolkit-Unity/blob/master/Assets/HoloToolkit/Sharing/Scripts/VoiceChat/MicrophoneReceiver.cs#L258)

Immediately following the audio source direction in the stream is the audio buffer.  I think this means the audio data is read incorrectly, it actually takes the first 12 bytes from the direction vector and misses the last 12 bytes.

Fix is either to not add it to the stream, or to decode it and ignore it (or skip the stream forward by 12 bytes).

### PR DESCRIPTION
**Pull request by StephenHodgson**
Fixes #786 
- Added HTRF bits to be read from the stream to prevent mismatch on receiving end.

### CODE REVIEW

File: Assets/HoloToolkit/Sharing/Scripts/VoiceChat/MicrophoneTransmitter.cs [hunk 1]
```diff
@@ -271,9 +271,9 @@ private void SendFixedSizedChunk(NetworkConnection connection, byte[] data, int
             msg.Write(cameraPosRelativeToGlobalAnchor.z);
 
             // HRTF direction bits
```
Comment (jessemcculloch) [inline Assets/HoloToolkit/Sharing/Scripts/VoiceChat/MicrophoneTransmitter.cs]: Is there a reason we are leaving this in commented out, vs just deleting the section?

Comment (StephenHodgson) [reply Assets/HoloToolkit/Sharing/Scripts/VoiceChat/MicrophoneTransmitter.cs]: Because potentially someone could implement the hrtf parts.

Comment (jessemcculloch) [reply Assets/HoloToolkit/Sharing/Scripts/VoiceChat/MicrophoneTransmitter.cs]: But the current implementation is wrong?  Sorry, i don't know much about this stuff.

Comment (StephenHodgson) [reply Assets/HoloToolkit/Sharing/Scripts/VoiceChat/MicrophoneTransmitter.cs]: Yeah it's causing a mismatch in the stream in the receiving end.

Comment (jessemcculloch) [reply Assets/HoloToolkit/Sharing/Scripts/VoiceChat/MicrophoneTransmitter.cs]: Got it.  Seems like maybe we should just put a TODO in on the Transmitter like in the Receiver.  Or put a comment in on why that code is commented out.

Personally I'm not a big fan of leaving commented code in.

Comment (StephenHodgson) [reply Assets/HoloToolkit/Sharing/Scripts/VoiceChat/MicrophoneTransmitter.cs]: I could comment it back in on the receiving end too, but I figured this way was saving network traffic.

**Pull request merged**